lib/ansible/plugins/action/__init__.py

```diff
@@ -754,9 +754,13 @@ def _execute_module(self, module_name=None, module_args=None, tmp=None, task_var
 
         # pre-split stdout/stderr into lines if needed
         if 'stdout' in data and 'stdout_lines' not in data:
-            data['stdout_lines'] = data.get('stdout', u'').splitlines()
+            # if the value is 'False', a default won't catch it.
+            txt = data.get('stdout', None) or u''
```

Comment:
I know, right? But if `data['stdout']=False`, then `data.get('stdout', u'')` returns `False`. That's why I added a comment- it isn't intuitive.

The next line is `txt.splitlines()`, which doesn't work on `False`.

I think this would also fail for `None`. It took some REPL work to make sure I understood what was happening.